ja/about/index.md

```diff
@@ -174,13 +171,6 @@ MRI以外のRuby処理系には以下のようなものがあります。
 * [mruby][mruby]はRubyの軽量実装です。アプリケーションにリンクし、組み込むことができます。Rubyの作者であるまつもと ゆきひろ氏によって開発が進められています。
 * [truffleruby][truffleruby]はGraalVM上に構築された高速なRuby実装です。
 
-#### 以下のバージョンはここ数年ほとんどまたは全く更新されていませんが、引き続き利用可能です。
-
-* [Rubinius][rubinius]は「Rubyで書かれたRuby」です。LLVM上に構築されています。LLVMは他の言語も構築されるよくできた仮想マシンです。
```

Comment:
I guess deletion of this line is not intentional, because no other translations in this pull request does the same thing.

Reply:
@shyouhei It was intentional. The other translations don't have that at all. They didn't separate frequently-updated from less-updated.

I removed that line because all the less-frequently-updated ones are totally dead now.

Reply:
OK, got it.  Then you might want also want to delete the line starts with `[rubinius]` below.  That would become an orphan link definiton.

Reply:
@shyouhei This is done.

Reply:
>  The other translations don't have that at all. They didn't separate frequently-updated from less-updated.

This isn't entirely true, see:

https://github.com/ruby/www.ruby-lang.org/blob/eb6223c62011b571b826bf84e6a72b217d36475e/en/about/index.md?plain=1#L189

and:

https://github.com/ruby/www.ruby-lang.org/blob/eb6223c62011b571b826bf84e6a72b217d36475e/ko/about/index.md?plain=1#L149